frontend/src/app/accountCreation/MfaSelect/MfaSelect.test.tsx

```diff
@@ -3,13 +3,27 @@ import { MemoryRouter, Route, Switch } from "react-router";
 
 import { MfaEmail } from "../MfaEmail/MfaEmail";
 import { MfaGoogleAuth } from "../MfaGoogleAuth/MfaGoogleAuth";
-import { MfaOktaVerify } from "../MfaOktaVerify/MfaOktaVerify";
+import { MfaOkta } from "../MfaOkta/MfaOkta";
 import { MfaPhone } from "../MfaPhone/MfaPhone";
 import { MfaSecurityKey } from "../MfaSecurityKey/MfaSecurityKey";
 import { MfaSms } from "../MfaSms/MfaSms";
 
 import { MfaSelect } from "./MfaSelect";
 
+jest.mock("../AccountCreationApiService", () => ({
+  AccountCreationApi: {
+    enrollTotpMfa: (app: string) => {
+      return new Promise((res, rej) => {
+        if (app === "Google" || app === "Okta") {
```

Comment:
Can `app` be a string union or enum rather than a string?

Reply:
Fixed in b732210